nextjs-13/src/app/auth/sign-in/page.test.tsx

```diff
@@ -1,40 +1,30 @@
 import { render, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { redirect } from 'next/navigation';
-import { useSession, signIn } from 'next-auth/react';
+import { signIn } from 'next-auth/react';
 
 import SignInPage from './page';
 
 jest.mock('next-auth/react');
 jest.mock('next/navigation');
 
 describe('SignInPage', () => {
-  describe('Session status is "authenticated', () => {
-    it('redirects to home page', () => {
-      useSession.mockReturnValue({ status: 'authenticated' });
-      render(<SignInPage />);
-      expect(redirect).toHaveBeenCalledWith('/');
-    });
+  it('renders h4', () => {
```

Comment:
It should be the following to be consistent with the other tests:

```suggestion
  it('renders the title', () => {
```

Reply:
Fixed [here](https://github.com/nimblehq/js-framework-benchmark/pull/123/commits/108cc7cf1e36665071c8d25ff89f06df610ed39c#diff-c54ff5d2e911b6e1771914307c60d6c0ca79b7f9d835e08eac133fdf4c7e98cbR11)